Resources/Computing/BatchSystems/SLURM.py

```diff
@@ -59,10 +63,23 @@ def submitJob(self, **kwargs):
       # By default, all the environment variables of the submitter node are propagated to the workers
       # It can create conflicts during the installation of the pilots
       # --export restricts the propagation to the PATH variable to get a clean environment in the workers
-      cmd += "sbatch --export=PATH -o %s/%%j.out --partition=%s -n %s %s %s " % (
-          outputDir, queue, numberOfProcessors, submitOptions, executable)
-      status, output = commands.getstatusoutput(cmd)
+      cmd += "sbatch --export=PATH "
+      cmd += "-o %s/%%j.out " % outputDir
+      cmd += "-e %s/%%j.err " % errorDir
+      cmd += "--partition=%s " % queue
+      # One pilot (task) per node, allocating a certain number of cpus
+      cmd += "--ntasks-per-node=1 "
+      cmd += "--nodes=%s " % numberOfNodes
+      cmd += "--cpus-per-task=%d " % numberOfProcessors
+      # Additional options
+      cmd += "%s %s" % (submitOptions, executable)
+      result = systemCall(30, cmd)
+      if not result['OK']:
+        status = -1
+        error = result['Message']
+        break
 
+      status, output, error = result['Value']
```

Comment:
When we are here, looks to me that `result` can still have `result['OK'] == False`.

Reply:
if `result['OK'] == False` there is a break, so it should never reach this line, am I wrong?